### ISSUE TITLE
Could you provide a link to download the training set?

### ISSUE THREAD

**Issue by yangco-le** [MaxiBoether/mis-benchmark-framework#1]
The training data generation procedure would take a lot of time if they have to be labeled from scratch by Gurobi. So could you provide a training set download link for followers to get a quick start?

**Comment by MaxiBoether** [comment]
Hey, thanks for reaching out. The core issue is that we do not own any rights to re-upload the graph data sets and we would need to get permission for each individual data set. I could check if we still have the random graphs we generated and their labels - however, for all non ER-graphs, Gurobi was able to label them instantaneously. Would these random graphs help you or are you specifically interested in the real-world data sets like SATLIB? Other than that, for us, the label generation on the real world graphs did not take too long. We ran it over night and it was done next morning.

Sorry I cannot help you directly here, but I do not want to run into copyright issues by re-uploading data I do not really have the rights to re-upload :/

**Comment by yangco-le** [comment]
Okay, I get it. I happen to be trying to generate the EG graphs, and it seems to be a bit time consuming. I am ok with the time you mentioned for real world graphs, thanks for the prompt reply!

**Comment by MaxiBoether** [comment]
Hi @yangco-le,

because we have received another request for this, I just wanted to let you know that we added our models & the labeled random graphs to the readme. You can download and use them, if you still need them.
